src/index.html

```diff
@@ -16,17 +17,29 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar search-bar--big"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input search-bar__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
The `placeholder` text should be concise and should not contain instructions like 'Try'. Consider shortening it to just 'Los Angeles' to improve the user experience.